modules/oadp-features.adoc

```diff
@@ -9,13 +9,26 @@
 OpenShift API for Data Protection (OADP) supports the following features:
 
 Backup::
-You can back up all resources in your cluster or you can filter the resources by type, namespace, or label.
+You can use OADP to back up all applications on the OpenShift Platform, or you can filter the resources by type, namespace, or label.
 +
 OADP backs up Kubernetes objects and internal images by saving them as an archive file on object storage. OADP backs up persistent volumes (PVs) by creating snapshots with the native cloud snapshot API or with the Container Storage Interface (CSI). For cloud providers that do not support snapshots, OADP backs up resources and PV data with Restic.
 
++
+[NOTE]
+====
+You must exclude Operators from an application, for backup and restore to succeed.
```

Comment:
s/You must exclude Operators from an application/You must exclude Operators from an application backup/g